ksql-engine/src/main/java/io/confluent/ksql/function/udf/string/InitCap.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.string;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.util.KsqlConstants;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@SuppressWarnings("unused") // Invoked via reflection.
+@UdfDescription(name = "initcap",
+    author = KsqlConstants.CONFLUENT_AUTHOR,
+    description = "Capitalizes the first letter of each word in a string and the rest lowercased.")
```

Comment:
a few words here, and in the syntax reference, about what constitutes a new "word" in the string would be appropriate - it's not as straightforward as one might hope with various unicode strings!